Package.swift

```diff
@@ -3,3 +3,5 @@ import PackageDescription
 let package = Package(
     name: "GeoSwift"
 )
+
+let x = 0 // trigger swiftlint
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/petrpavlik/GeoSwift/blob/39f3c59b1843ddba396fdb7110d365a9624987f6/Package.swift#L7"></span>Variable name should be between 3 and 40 characters long: ‘x’</td>
    </tr>
  </tbody>
</table>